lustrefs-exporter/src/jobstats.rs

```diff
@@ -134,6 +134,9 @@ pub fn jobstats_stream<R: BufRead + std::marker::Send + 'static>(
     let x = tokio::task::spawn_blocking(move || {
         let mut state = State::Empty;
 
+        // Send a new line to make sure we are printing stats with a separating empty line
+        _ = tx.blocking_send("\n".to_compact_string());
```

Comment:
Is this needed? Do our other stats end with a newline char?

Reply:
It's not needed, it helps readability by adding a new line for jobstats related metric.